.github/workflows/CI-jython.yml

```diff
@@ -26,6 +26,9 @@ jobs:
           jython/bin/jython -m easy_install robotframework==4.1.3 robotstatuschecker==1.5.1 robotbackgroundlogger
       - name: Configure test environment for Linux
         run: |
+          sudo useradd --create-home --shell /bin/bash test-nopasswd
+          sudo passwd --delete test-nopasswd
+          echo "PasswordAuthentication yes" | sudo tee -a /etc/ssh/sshd_config
```

Comment:
Wouldn't be better to add `PermitEmptyPasswords yes`? Password authentication is already enabled since we have the `test` user with `test` password.

Reply:
Yes you are right, thank you!